### Modified copy of kinematics_msgs/GetPositionFK.srv - used since arm_navigation hasn't been catkinized yet! ###
# A service definition for a standard forward kinematics service
# The frame_id in the header message is the frame in which 
# the forward kinematics poses will be returned
Header header

# A vector of link name for which forward kinematics must be computed
string[] fk_link_names

# A robot state consisting of joint names and joint positions to be used for forward kinematics
sensor_msgs/JointState robot_state
---
# The resultant vector of PoseStamped messages that contains the (stamped) poses of the requested links
geometry_msgs/PoseStamped[] pose_stamped

# The list of link names corresponding to the poses
string[] fk_link_names

#arm_navigation_msgs/ArmNavigationErrorCodes error_code - replace with own error codes
